engine/api/workflow_run.go

```diff
@@ -451,6 +451,10 @@ func (api *API) postWorkflowRunHandler() Handler {
 			return sdk.WrapError(err, "postWorkflowRunHandler> Unable to commit transaction")
 		}
 
+		// Purge workflow run
+		branch := workflow.GetTag(wr.Tags, "git.branch")
```

Comment:
Should it be a part of a workflow deletion strategy ? To be able to delete with other criteria